src/Components/Setting/Appearance/Appearance.ts

```diff
@@ -127,16 +139,13 @@ const Appearance = async (): Promise<void> => {
 	</div>
 	<h3 class="settings-title">${transparentEffect_i18n}</h3>
 	<select name="transparent-effect">
-		<option value="blur" ${transparentEffect === 'blur' ? 'selected' : ''} ${platform !== 'win32' ? 'disabled' : ''} ${disabledForWeb}>Blur</option>
-		<option value="acrylic" ${transparentEffect === 'acrylic' ? 'selected' : ''} ${
-		platform !== 'win32' ? 'disabled' : ''
-	} ${disabledForWeb}>Acrylic</option>
-		<option value="mica" ${transparentEffect === 'mica' ? 'selected' : ''} ${
-		!(platform === 'win32' && (await isWin11())) ? 'disabled' : ''
-	} ${disabledForWeb}>Mica</option>
-		<option value="vibrancy" ${transparentEffect === 'vibrancy' ? 'selected' : ''} ${
-		platform !== 'darwin' ? 'disabled' : ''
-	} ${disabledForWeb}>Vibrancy</option>
+		<option value="blur" ${transparentEffect === 'blur' ? 'selected' : ''} ${platform !== 'win32' ? 'disabled' : ''} ${disabledForWeb}>${blur_i18n}</option>
+		<option value="acrylic" ${transparentEffect === 'acrylic' ? 'selected' : ''} ${platform !== 'win32' ? 'disabled' : ''
+		} ${disabledForWeb}>${acrylic_i18n}</option>
+		<option value="mica" ${transparentEffect === 'mica' ? 'selected' : ''} ${!(platform === 'win32' && (await isWin11())) ? 'disabled' : ''
+		} ${disabledForWeb}>${mica_i18n}</option>
+		<option value="vibrancy" ${transparentEffect === 'vibrancy' ? 'selected' : ''} ${platform !== 'darwin' ? 'disabled' : ''
+		} ${disabledForWeb}>${vibrancy_i18n}</option>
 		<option value="none" ${transparentEffect === 'none' ? 'selected' : ''}>None</option>
```

Comment:
I think you forgot to change this one too, could you please edit it? hehe

Reply:
Oh... yes!
It was in my local repo, but when I was cleaning stuff up for the PR it somehow got lost